### ISSUE TITLE
Http error 500

### ISSUE THREAD

**Issue by josephsdavid** [cachix/cachix#214]
I can’t do anything, use any of my nix shells or even uninstall cachix because cachix is down

**Comment by josephsdavid** [comment]
Can I like do cachix don’t use so I can at least do some work while cachix is down

**Comment by josephsdavid** [comment]
Sorry that came across as rude, but is there a way i can tell cachix to not use a cache when cachix is down? That way I am not in trouble when this happens again

**Comment by doronbehar** [comment]
I got the same behavior today when running `nixos-rebuild`. Also using the `--option substituters` didn't help I guess because the relevant Nix options were written in `/etc/nix/nix.conf` - it still tried to use the cachix servers I've had configured.

I've eventually worked around this by disabled these substitutes manually:
- I deleted `/etc/nix` which is a symlink to a read only file system in the store.
- I recreated that directory.
- I wrote the same `nix.conf` file there only without the problematic substitutes.
